README.md

```diff
@@ -41,6 +42,15 @@ Run server:
 
 Then go to `http://127.0.0.1:3000` to view the locally running application.
 
+#### Docker:
+
+Alternatively, you can create a Docker image and start it in a container:
+```
+docker build -t health_cards .
+docker run -p 3000:3000 health_cards
```

Comment:
I should have added this in my PR, but can you update this to include the necessary environment variable:

```suggestion
docker run -p 3000:3000 --env HOST=http://localhost:3000 health_cards
```

Reply:
I added this to the command and then added a sentence right after instructing issuers to pass in their own personal HOST environment variable.